helm/v1-authors-transformer/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for v1-authors-transformer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+service:
+  name: "" # The name of the service, should be defined in the specific app-configs folder.
+  hasHealthcheck: "true"
+replicaCount: 1
+image:
+  repository: coco/v1-authors-transformer
+  pullPolicy: IfNotPresent
+resources:
+  limits:
+    memory: 1Gi
```

Comment:
Does the memory limit really need to be 1Gi? According to grafana average memory load is below 20Mi. Even when there were a few cpu usage spikes there hasn't been much of an increase in memory usage.

Reply:
Yeah I had a look at this too, that's why I set the request low - the top limit is just to determine when it's killed with an OOM